### PR TITLE
Improves the UX around closing issues on fullscreen issue view

### PR DESCRIPTION
**Pull request by discorick**
- Changes copy
- adds a close button to all modal windows

image

### CODE REVIEW

**Comment by rauhryan** [comment]
I'd like to see a version where it's a big pretty white X in the very top right

window_and_huboard_huboard_huboard_and_iphoto

**Comment by discorick** [comment]
Can do, before I do that, its worth noting this PR gives EVERY modal a closing "x" (which seems like a win). Not all of our modals dim the background like a fullscreen issue, so we would need to roll it as a "special snowflake" or provide an override in the modal view.

**Comment by rauhryan** [comment]
Please list out all our modals; (screenshots if possible)

Which one's dim and which do not dim?

Perhaps the one's that don't dim are the "special snowflake" and not the other way around

**Comment by discorick** [comment]
Weird... I looked through all the modals, the DO in fact dim out the background. Not sure where I got the impression otherwise. I will spike out the bold white X in the top right.

**Comment by discorick** [comment]
image

@rauhryan what do you think?

**Comment by rauhryan** [comment]
Yes, I approve

File: app/assets/stylesheets/_overlay.scss [hunk 1]
```diff
@@ -77,7 +77,14 @@ body > .fullscreen-overlay {
       }
     }
   }
-
+  .ui-icon-x-thin {
```
Comment (rauhryan) [inline app/assets/stylesheets/_overlay.scss]: scope this to something else, we might need to use that icon for other things within an overlay

File: ember-app/app/views/modal.js [hunk 2]
```diff
@@ -36,14 +36,22 @@ var ModalView = Ember.View.extend({
...
-       this.get('controller').send('closeModal');
-     }
+     this.close();
     }.bind(this));
 
     this.$(':input:not(.close):not([type="checkbox"])').first().focus();
+
+    this.$('.modal-close').click(function(){
+      this.close();
+    }.bind(this));
```
Comment (rauhryan) [inline ember-app/app/views/modal.js]: Not ideal here, I think this should be an action... 

But for now, please add the unsubscribe event in the destroyElement hook

**Comment by rauhryan** [comment]
:shipit:

**Pull request merged**